lib/pharos/config.rb

```diff
@@ -37,6 +37,11 @@ def self.load(raw_data)
       config.hosts.each do |host|
         host.api_endpoint = config.api&.endpoint
         host.config = config
+        # map bastion.host to an existing host
```

Comment:
How this is related?

Reply:
Otherwise the `bastion.host` points to a duplicate created via `Configuration::Host.new` instead to the actual host in config.

This means, that for example `host.ssh.gateway.shutdown!` when doing a host reboot/reconnect will shut down a gateway that is not the duplicated bastion host's gateway.